lib/tasks/set_missions_to_regulated.rake

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# This namespace contains all tasks related to the mission model
+namespace :mission do # rubocop:disable Metrics/BlockLength
+  desc "set futur mission with the :name 'permanence clac' to :regulated"
+  task set_missions_to_regulated: :environment do
+    missions = Mission.where('start_date > ? and genre = ? and name = ?',
+                             DateTime.current + 1.day,
+                             0,
+                             'permanence clac')
```

Comment:
If you think that checking case-insensively could be useful, you can use pgsql's `LOWER()` method in the `.where` query string for the `name`